upf/pfcp.c

```diff
@@ -86,34 +86,38 @@ format_enum (u8 * s, va_list * args)
 }
 
 u8 *
-format_network_instance (u8 * s, va_list * args)
+format_apn_string (u8 * s, va_list * args)
```

Comment:
Renamed `format_apn_string()` to `format_dns_labels()` for now. It may be that we also need to support "plaintext" format for the network instance IE but that should probably be done separately